code/game/machinery/doors/windowdoor.dm

```diff
@@ -317,6 +317,14 @@
 /obj/machinery/door/window/narsie_act()
 	add_atom_colour(NARSIE_WINDOW_COLOUR, FIXED_COLOUR_PRIORITY)
 
+/obj/machinery/door/window/rust_heretic_act()
+	add_atom_colour("#917c65", FIXED_COLOUR_PRIORITY)
+	set_armor(/datum/armor/none)
+	take_damage(get_integrity() * 0.5)
+	modify_max_integrity(max_integrity * 0.5)
+	if(!HAS_TRAIT(src, TRAIT_RUSTY))
+		AddElement(/datum/element/rust)
```

Comment:
Why are we applying the rust element to the windoor anyways, if we're adding an atom color to it? One or the other (probably just the color)